common/djangoapps/student/tests/test_helpers.py

```diff
@@ -0,0 +1,33 @@
+""" Test Student helpers """
+
+import logging
+
+from django.test import TestCase
+from django.test.client import RequestFactory
+from testfixtures import LogCapture
+
+from student.helpers import get_next_url_for_login_page
+
+
+LOGGER_NAME = "student.helpers"
+
+
+class TestLoginHelper(TestCase):
+    """Test login helper methods."""
+    def setUp(self):
+        super(TestLoginHelper, self).setUp()
+        self.request = RequestFactory()
+
+    def test_unsafe_next(self):
+        """ Test unsafe next parameter """
+        with LogCapture(LOGGER_NAME, level=logging.ERROR) as logger:
+            req = self.request.get("http://testserver/login?next=http://amazon.com")
+            get_next_url_for_login_page(req)
+            logger.check(
+                (LOGGER_NAME, "ERROR", u"Unsafe redirect parameter detected: u'http://amazon.com'"))
```

Comment:
Maybe you should assign a variable with value `https://www.amazon.com` so you don't have to repeat the value here